db/db/src/main/java/io/helidon/db/DbExecute.java

```diff
@@ -0,0 +1,407 @@
+        return createNamedGet(statementName).params(parameters).execute();
+    }
+
+    /**
+     * Create and execute a database query using a statement passed as an argument.
+     *
+     * @param statement  the query statement to be executed
+     * @param parameters query parameters to set
+     * @return database query execution result which can contain single row
+     */
+    default CompletionStage<Optional<DbRow>> get(String statement, Object... parameters) {
+        return createGet(statement).params(parameters).execute();
+    }
+
+    /*
+     * INSERT
+     */
+
+    /**
+     * Create an insert statement using a named statement passed as an argument.
+     *
+     * @param statementName the name of the statement
+     * @param statement the statement text
+     * @return database statement that can insert data
+     */
+    default DbStatement<?, CompletionStage<Long>> createNamedInsert(String statementName, String statement) {
+        return createNamedDmlStatement(statementName, statement);
+    }
+
+    /**
+     * Create an insert statement using a named statement.
+     *
+     * @param statementName the name of the statement
+     * @return database statement that can insert data
+     */
+    DbStatement<?, CompletionStage<Long>> createNamedInsert(String statementName);
+
+    /**
+     * Create an insert statement using a statement text.
+     *
+     * @param statement the statement text
+     * @return database statement that can insert data
+     */
+    DbStatement<?, CompletionStage<Long>> createInsert(String statement);
+
+    /**
+     * Create and execute insert statement using a statement defined in the configuration file.
+     *
+     * @param statementName the name of the configuration node with statement
+     * @param parameters    query parameters to set
+     * @return number of rows inserted into the database
+     */
+    default CompletionStage<Long> namedInsert(String statementName, Object... parameters) {
+        return createNamedInsert(statementName).params(parameters).execute();
+    }
+
+    /**
+     * Create and execute insert statement using a statement passed as an argument.
+     *
+     * @param statement  the insert statement to be executed
+     * @param parameters query parameters to set
+     * @return number of rows inserted into the database
+     */
+    default CompletionStage<Long> insert(String statement, Object... parameters) {
+        return createInsert(statement).params(parameters).execute();
+    }
+
+    /*
+     * UPDATE
+     */
+
+    /**
+     * Create an update statement using a named statement passed as an argument.
+     *
+     * @param statementName the name of the statement
+     * @param statement the statement text
+     * @return database statement that can update data
+     */
+    default DbStatement<?, CompletionStage<Long>> createNamedUpdate(String statementName, String statement) {
```

Comment:
Another occurence of _hybrid_ method with both statement name and statement string.